### PR TITLE
test: manifest and poa starship tests

### PR DESCRIPTION
**Pull request by fmorency**
To run,

```shell
yarn starship start && yarn starship:test && yarn starship stop
```

Make sure your Starship environment is properly set following the guide in https://docs.cosmology.zone/starship/get-started/step-1

### CODE REVIEW

**Comment by fmorency** [comment]
Still working on this.

**Comment by chalabi2** [comment]
I keep getting 

```
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/chalabi/Code/Cosmos/manifestjs/node_modules/strip-ansi/index.js from /Users/chalabi/Code/Cosmos/manifestjs/node_modules/cliui/build/index.cjs not supported.
Instead change the require of index.js in /Users/chalabi/Code/Cosmos/manifestjs/node_modules/cliui/build/index.cjs to a dynamic import() which is available in all CommonJS modules.
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at Object.<anonymous> (/Users/chalabi/Code/Cosmos/manifestjs/node_modules/cliui/build/index.cjs:292:19)
```

when running `yarn starship:test `

**Comment by fmorency** [comment]
> I keep getting
> 
> ```
> Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/chalabi/Code/Cosmos/manifestjs/node_modules/strip-ansi/index.js from /Users/chalabi/Code/Cosmos/manifestjs/node_modules/cliui/build/index.cjs not supported.
> Instead change the require of index.js in /Users/chalabi/Code/Cosmos/manifestjs/node_modules/cliui/build/index.cjs to a dynamic import() which is available in all CommonJS modules.
>     at TracingChannel.traceSync (node:diagnostics_channel:315:14)
>     at Object.<anonymous> (/Users/chalabi/Code/Cosmos/manifestjs/node_modules/cliui/build/index.cjs:292:19)
> ```
> 
> when running `yarn starship:test `

No idea. Are you able to run simple starship examples from their repo?

File: starship/src/test_helper.ts [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+const submitGroupProposal = async (
+  signer: string,
+  client: SigningStargateClient,
+  title: string,
+  summary: string,
+  proposers: string[],
+  messages: { typeUrl: string; value: Uint8Array }[], // Encoded messages
+  fee: { amount: { denom: string; amount: string }[]; gas: string }
+) => {
+  // COMMENTING THE FOLLOWING LINE WILL CAUSE THE PAYOUT/BURN TEST TO FAIL. WHY?
```
Comment (fmorency) [inline starship/src/test_helper.ts:95]: Why?

**Pull request merged**